lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -71,8 +72,28 @@ def add_arguments(self, parser):
             default=0,
             type=int,
         )
+        parser.add_argument(
+            '--estimate_first_attempted',
+            help='Use score data to estimate first_attempted timestamp. Values: (true|false|none)',
+            default='none',
+            choices={'true', 'false', 'none', 'True', 'False', 'None'}
```

Comment:
I think it's fair to assume that the default setting will be used if it isn't explicitly overridden in the command, so you could just allow (true|false). I also like (true|false|default). Using `None` just feels like, given what passing `True` or `False` does, it would set the waffle flag to `None`, not use the default.